bundles/org.openhab.binding.upnpcontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -257,13 +289,21 @@
 		<item-type>String</item-type>
 		<label>Current Media Id</label>
 		<description>Current id of media entry or container, option list to browse hierarchy</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>MediaControl</tag>
```

Comment:
```suggestion
			<tag>Channel</tag>
```

Reply:
I will handle this in https://github.com/openhab/openhab-addons/pull/18544